packages/js/manifests/polywrap/src/formats/polywrap.plugin/0.2.0.ts

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+/* tslint:disable */
+/**
+ * This file was automatically generated by json-schema-to-typescript.
+ * DO NOT MODIFY IT BY HAND. Instead, modify the source JSONSchema file,
+ * and run json-schema-to-typescript to regenerate this file.
+ */
+
+export interface PluginManifest {
+  /**
+   * Polywrap manifest format version.
+   */
+  format: "0.2.0";
+  /**
+   * Basic project properties.
+   */
+  project: {
+    /**
+     * Name of this project.
+     */
+    name: string;
+    /**
+     * Type of this project.
+     */
+    type: string;
+  };
+  /**
+   * Project source files.
+   */
+  source: {
+    /**
+     * Path to the project's entry point.
+     */
+    module?: string;
```

Comment:
`module` should always be defined IMO; otherwise, it would be an interface right?

Reply:
You are correct, I missed that one.